app/assets/stylesheets/common/foundation/base.scss

```diff
@@ -13,6 +13,14 @@ html {
   background-color: $secondary;
   overflow-y: scroll;
   direction: ltr;
+
+  &.font-size-larger {
+    font-size: $base-font-size-larger;
+  }
+
+  &.font-size-largest {
+    font-size: $base-font-size-largest;
+  }
```

Comment:
@awesomerobot are you happy for these dynamic sizes to be in `foundation/base.scss`, or would you rather they were moved somewhere else?